src/test/java/nextstep/cucumber/steps/PathStepDef.java

```diff
@@ -107,6 +107,13 @@ public PathStepDef() {
             assertThat(expectDistance).isEqualTo(Integer.valueOf(distance));
             assertThat(expectDuration).isEqualTo(Integer.valueOf(duration));
         });
+
+        Then("지하철 이용 요금 {string}를 리턴한다", (String fee) -> {
+            final PathResponse pathResponse = context.response.as(PathResponse.class);
+            final int expectDistance = pathResponse.getFare();
+
+            assertThat(expectDistance).isEqualTo(Integer.valueOf(fee));
+        });
```

Comment:
int 타입을 활용해 보셔도 좋을 것 같아요 😃 
```suggestion
        Then("지하철 이용 요금 {int}를 리턴한다", (Integer fee) -> {
            final PathResponse pathResponse = context.response.as(PathResponse.class);
            final int expectDistance = pathResponse.getFare();

            assertThat(expectDistance).isEqualTo(fee);
        });
```